frappe/installer.py

```diff
@@ -152,7 +155,15 @@ def install_db(
 
 	frappe.flags.root_login = root_login
 	frappe.flags.root_password = root_password
-	setup_database(force, source_sql, verbose, no_mariadb_socket)
+
+	if setup:
+		setup_database(force, source_sql, verbose, no_mariadb_socket)
```

Comment:
Can drop this parameter now, its unused there.
```suggestion
		setup_database(force, verbose, no_mariadb_socket)
```

Reply:
https://github.com/frappe/frappe/pull/23170/commits/0dc2a9c2f39d54f970c340362e93cdd0c594b0ff